server/issue.go

```diff
@@ -630,7 +630,7 @@ func (p *Plugin) assignJiraIssue(mmUserId, issueKey, assignee string) (string, e
 	}
 
 	// Get list of assignable assignees
-	url := fmt.Sprintf("rest/api/3/user/assignable/search?issueKey=%s&query=%s", issueKey, assignee)
+	url := fmt.Sprintf("rest/api/2/user/assignable/search?issueKey=%s&username=%s", issueKey, assignee)
```

Comment:
Note the change here.  Jira `Server` `api/3` does not exist, so I moved the api call to `api/2`.  However, this means the `query` parameter won't exist and we have to use the `username` parameter.

Reply:
Just curious about how this was able to work before?

Reply:
This was working because of testing done on mmtest and I can only assume server testing was skipped.